sdks/python/scripts/run_pylint.sh

```diff
@@ -60,8 +60,15 @@ EXCLUDED_GENERATED_FILES=(
 apache_beam/portability/api/*pb2*.py
 )
 
+# Following files contain python 3 syntax and are excluded from pylint in python 2
+EXCLUDED_PYTHON3_FILES=(
+"apache_beam/transforms/transforms_keyword_only_args_py3_test.py"
```

Comment:
I think  `_py3_only_test` may be a more explicit suffix, how about we use that?
Also could this be a regex pattern `.*_py3_only_test.py`?